src/contexts/RefinebioContext.js

```diff
@@ -22,6 +22,7 @@ export const RefinebioContextProvider = ({ children }) => {
     'requested-experiments',
     []
   )
+  const [samplesTableData, setSamplesTableData] = useState({})
```

Comment:
I'm having trouble understanding why we should put this data here. This is our global application context and should be used for data that needs to be persisted and accessed in multiple locations.

Reply:
The `samplesTableData` was previously shared with `src/pages/experiments/[accession_code]/[experiment]/index` to calculate the correct scroll position of the samples table scroll animation (ensuring the table data was fully loaded). But it has been moved from the global context and is defined in `SamplesTableContext` (at cee5282ac17f02756443223a717fb6cb5a07cdd5).

Instead, I am now using the [ResizeObserver API](https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver) to handle the calculation for the samples table position on layout changes (at f5edd0278fd6caed0e1b947db07e8f9b6e8b3c26).

I've also made a small revision to `helpers/scrollTop` (previously was named `scrollToTop`) and adjusted the codebase (at ec71236b7c3577d370dff6239eb7e0f9f4aa2f2f). Thank you, David!